src/EnergyPlus/CMakeLists.txt

```diff
@@ -612,3 +612,49 @@ if(UNIX AND NOT APPLE)
   COMMAND ${CMAKE_COMMAND} -E create_symlink energyplus EnergyPlus
   COMMAND ${CMAKE_COMMAND} -E rename EnergyPlus "${CMAKE_BINARY_DIR}/Products/EnergyPlus") 
 endif()
+
+# if we are building a package, we can do additional operations, such as run doc scripts
+if(BUILD_PACKAGE)
+  
+  # Some docs are generated on the fly here, create a dir for the 'built' files
+  set( DOCS_OUT "${CMAKE_BINARY_DIR}/autodocs" )
+  file( MAKE_DIRECTORY "${DOCS_OUT}" )
+  
+  # the output variables listing
+  add_custom_command( 
```

Comment:
You can probably just make it depend on ```Target package``` instead of ```Target energyplus```.  But even that feels a little off to me.  Maybe better to have the dependency setup on the files that it is actually dependent on.  In this case the example files.  I can play with it, but if you have ideas for a better dependency chain I'm invested in that.  I'm pretty sure energyplus is not it.